samples/repo-search/desktop-app/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
```

Comment:
IDK what I am doing wrong but I couldn't get the desktop-app module to build without explicitly adding a grade wrapper + properties. It _should_ just work with picking up the project level ones but.. idk?

Reply:
After deleting this `gradle` directory, I ran `./gradlew samples:repo-search:desktop-app:run` and it worked. How were you running it?

Reply:
Hmm I was running it through the run configuration that was picked up automatically. I'll check this again and remove it next commit